controllers/machine_controller.go

```diff
@@ -537,18 +538,18 @@ func (r *MachineReconciler) drainNode(ctx context.Context, cluster *clusterv1.Cl
 
 	if err := kubedrain.RunCordonOrUncordon(ctx, drainer, node, true); err != nil {
 		// Machine will be re-reconciled after a cordon failure.
-		log.Error(err, "Cordon failed")
-		return errors.Errorf("unable to cordon node %s: %v", node.Name, err)
+		log.Error(err, "Cordon failed, retry in 20s")
+		return ctrl.Result{}, errors.Errorf("unable to cordon node %s: %v", node.Name, err)
```

Comment:
The log line says retry in 20s, but we're returning an error which is going to make the controller go in exponential backoff